scripts/test_argument_options.sh

```diff
@@ -74,13 +74,12 @@ expect_usage_error -t gmander
 LOG_FILE="/tmp/`uuidgen`"
 printf "When given bad input, prints file not found... "
 $BITSHUFFLE --input /nonexistent/nope > "$LOG_FILE" 2>&1
-if [ "`cat $LOG_FILE`" = "Error: Input file not found" ]; then
+if grep -i 'could not open' < "$LOG_FILE" > /dev/null 2>&1 ; then
```

Comment:
Are we certain `grep -i` is portable? It works fine on my machine but I'd like to be sure.

Reply:
I have never seen a machine that implements `grep` but not `grep -i`. IIRC it's a pretty old feature for grep. I know for sure that it is present on macOS, Linux, and FreeBSD.